src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -75,46 +75,43 @@ async Task<RunResult> PerformTestRun()
 
         async Task PerformScenarios(ComponentRunner[] runners)
         {
-            using (var cts = new CancellationTokenSource())
+            try
```

Comment:
this method just looks noisy because I removed the outer CTS which didn't seem useful